examples/bigcommerce/README.md

```diff
@@ -31,69 +31,24 @@ BIGCOMMERCE_ACCESS_TOKEN=5lw9ulikcp186tjgg3rs39kh4fg3vci
 
 ---
 
-## Using this repo
+Here is [a guide](https://support.bigcommerce.com/s/article/Starting-a-BigCommerce-Trial) to get you started.
```

Comment:
This line seems out of place, since you reference it late.

Reply:
Good catch